[2025-09-19T11:16:07.621+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-19T11:16:07.653+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mayo_pipeline.etl_vitals manual__2025-09-19T11:16:07.151018+00:00 [queued]>
[2025-09-19T11:16:07.661+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mayo_pipeline.etl_vitals manual__2025-09-19T11:16:07.151018+00:00 [queued]>
[2025-09-19T11:16:07.662+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-09-19T11:16:07.676+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): etl_vitals> on 2025-09-19 11:16:07.151018+00:00
[2025-09-19T11:16:07.681+0000] {standard_task_runner.py:63} INFO - Started process 367 to run task
[2025-09-19T11:16:07.684+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'mayo_pipeline', 'etl_vitals', 'manual__2025-09-19T11:16:07.151018+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/mayo_pipeline.py', '--cfg-path', '/tmp/tmp04ojc5mx']
[2025-09-19T11:16:07.688+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask etl_vitals
[2025-09-19T11:16:07.755+0000] {task_command.py:426} INFO - Running <TaskInstance: mayo_pipeline.etl_vitals manual__2025-09-19T11:16:07.151018+00:00 [running]> on host d93916f05490
[2025-09-19T11:16:08.012+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='you' AIRFLOW_CTX_DAG_ID='mayo_pipeline' AIRFLOW_CTX_TASK_ID='etl_vitals' AIRFLOW_CTX_EXECUTION_DATE='2025-09-19T11:16:07.151018+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-19T11:16:07.151018+00:00'
[2025-09-19T11:16:08.013+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-19T11:16:08.043+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-19T11:16:08.044+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/project && python etl/flatten_mimic_vitals.py']
[2025-09-19T11:16:08.055+0000] {subprocess.py:86} INFO - Output:
[2025-09-19T11:16:08.057+0000] {subprocess.py:93} INFO - /usr/bin/bash: line 1: cd: /opt/project: No such file or directory
[2025-09-19T11:16:08.058+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2025-09-19T11:16:08.059+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-19T11:16:08.069+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-09-19T11:16:08.073+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=mayo_pipeline, task_id=etl_vitals, run_id=manual__2025-09-19T11:16:07.151018+00:00, execution_date=20250919T111607, start_date=20250919T111607, end_date=20250919T111608
[2025-09-19T11:16:08.084+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 3 for task etl_vitals (Bash command failed. The command returned a non-zero exit code 1.; 367)
[2025-09-19T11:16:08.103+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-09-19T11:16:08.125+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-19T11:16:08.127+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
